amanda_keogh/.jshintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "node":true,
+  "globals": {
+    "describe":true,
+    "it":true
+  }
+}
```

Comment:
This is a really small file compared to what I'm used to. Check out a ton of the options you can set for jshint: https://github.com/jshint/jshint/blob/master/examples/.jshintrc.